### ISSUE TITLE
Running insert_worm.py fails

### ISSUE THREAD

**Issue by travs** [openworm/PyOpenWorm#77]
This is my full stack trace when I run the script on both the `alpha0.5` and `alpha0.5-slarson` branches.
The error occurs in the step `update_neurons_and_muscles_with_lineage_and_descriptions()`.

```
└─▪ python insert_worm.py 
uploaded muscles
uploaded lineage and descriptions
uploaded synapses
uploaded receptors and innexins
Traceback (most recent call last):
  File "insert_worm.py", line 277, in do_insert
    update_neurons_and_muscles_with_lineage_and_descriptions()
  File "insert_worm.py", line 100, in update_neurons_and_muscles_with_lineage_and_descriptions
    cells = {next(x.name()) : (norn(x.lineageName()), norn(x.description())) for x in P.Cell().load() }
  File "insert_worm.py", line 100, in <dictcomp>
    cells = {next(x.name()) : (norn(x.lineageName()), norn(x.description())) for x in P.Cell().load() }
  File "insert_worm.py", line 92, in norn
    return next(x)
TypeError: unicode object is not an iterator
```

**Comment by mwatts15** [comment]
Hi @travs. Could you have a look at #76? In that pull request, `update_neurons_and_muscles_with_lineage_and_descriptions` is no longer called since the relevant activity takes place in `upload_lineage_and_descriptions`

**Comment by slarson** [comment]
Looks like this is fixed by #76 !
